docs: add SELinux note in prerequisites (#93)
khuedoan/homelab: docs/installation/production/prerequisites.md

# Prerequisites

## Fork this repository

Because [this project](https://github.com/khuedoan/homelab) applies GitOps practices,
it's the source of truth for _my_ homelab, so you'll need to fork it to make it yours:

[:fontawesome-solid-code-fork: Fork khuedoan/homelab](https://github.com/khuedoan/homelab/fork){ .md-button }

By using this project you agree to [the license](/license).

!!! summary "License TL;DR"

     - This project is free to use for any purpose, but it comes with no warranty
     - You must use the same [GPLv3 license](https://www.gnu.org/licenses/gpl-3.0.en.html)  in `LICENSE.md`
     - You must keep the copy right notice and/or include an acknowledgement
     - Your project must remain open-source

## Hardware requirements

### Initial controller

!!! info

    The initial controller is the machine used to bootstrap the cluster, we only need it once, you can use your laptop or desktop

- A Linux machine that can run Docker (because the `host` networking driver used for PXE boot [only supports Linux](https://docs.docker.com/network/host/), you can use a Linux virtual machine with bridged networking if you're on macOS or Windows).

> - Many linux distributions, such as Fedora, deploy with SELinux enabled and enforced. Configure this appropriately on your own or simply set to permissive (at least it'll be logging):
> ```
> sed -i s/^SELINUX=.*$/SELINUX=permissive/ /etc/selinux/config
> setenforce 0
> ```

Reviewer: ```suggestion
- Some Linux distributions, such as Fedora, deploy with SELinux enabled and enforced. Configure this appropriately on your own or simply [set it to permissive](https://docs.fedoraproject.org/en-US/quick-docs/changing-selinux-states-and-modes/#selinux-changing-to-permissive-mode).
```

We should link this to the official document instead, which contains more information and context.
